File: src/main/java/com/harleylizard/magic_things/common/MagicThingsBiomeTags.java
package com.harleylizard.magic_things.common;

import net.minecraft.core.registries.Registries;
import net.minecraft.tags.TagKey;
import net.minecraft.world.level.biome.Biome;

public final class MagicThingsBiomeTags {
    public static final TagKey<Biome> FOULED = TagKey.create(Registries.BIOME, MagicThings.resourceLocation("fouled"));

}

File: src/main/kotlin/com/harleylizard/magic_things/common/MagicThingsBiomeTags.kt
package com.harleylizard.magic_things.common

import com.harleylizard.magic_things.common.MagicThings.Companion.resourceLocation
import net.minecraft.core.registries.Registries
import net.minecraft.tags.TagKey
import net.minecraft.world.level.biome.Biome

object MagicThingsBiomeTags {
    val fouled: TagKey<Biome> = TagKey.create(Registries.BIOME, "fouled".resourceLocation)

}